### ISSUE TITLE
Ligature for common math symbols

### ISSUE THREAD

**Issue by bhuynhdev** [rsms/inter#457]
This is one of my favorite fonts and I love that it has ligatures for sequences like "<==" or "->". However, I often take notes with Math symbols such as ">=", "<=", and "!=" and I really hope Inter can support these ligatures too. Or if anyone can tell me a way to do that I will be happy to research it out.

Examples of how they may look
![image](https://user-images.githubusercontent.com/55563202/168317279-38fcc200-358e-4f0b-8e67-cf3fbd029ed7.png)

**Comment by rsms** [comment]
Maybe as an opt-in feature, but not by default.

**Comment by LoganDark** [comment]
Yes! Start the "Inter as a programming font" revolution! Monospaced Inter!

(JK. Inter is amazing, we already have so much for free and I couldn't imagine asking for a monospaced font too)

**Comment by Porges** [comment]
Look into using XCompose or WinCompose, then you can type <kbd>[Compose]>=</kbd> and have a real `≥` produced.

**Comment by Kajiih** [comment]
Is there a way to enable these ligatures, today?
